ONSdigital/sdc-responses-dashboard
# Add frontend config and associate ssl policy

# What and why?
This PR introduces a frontend config to responses-dashboard and associates it with the ssl policy created in the terraform update. 
# How to test?
Not the easiest to test due to lack of loadbalancer outside pre-prod and PROD. Steps to test through dev environment:
- Deleted existing deployment and service from environment
- Created temporary ip address `gcloud compute addresses create dashboard-ip --global` (deleted after testing).
- Manually created [terraform](https://github.com/ONSdigital/ras-rm-terraform/pull/355/files) equivalent SSL policy associated with the ingress. 
- Use helm to deploy the pr version of responses-dashboard
          helm upgrade --install responses-dashboard _infra/helm/responses-dashboard --namespace "" \
           --set-string gcp.project="ras-rm-dev" \
           --set-string env="dev" \
           --set-string namespace="" \
           --set-string image.tag="pr-151"
- Cloud armor policy is required for the ingress, manually created basic deny rule through ui (deleted after testing).

The updated policy should then be associated to the frontend ingress.

# Jira
https://jira.ons.gov.uk/browse/RAS-1064
